ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Get_Vendor_Files/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+##############
+# VENDOR_FILES
+##############
+
+- name: check out AdoptOpenJDK/secrets
+  local_action: git repo=git@github.com:AdoptOpenJDK/secrets.git dest=vendor_files
+  register: git_clone
+
+- name: check if dotgpg is installed
+  local_action: command command -v dotgpg
```

Comment:
Is there meant to be 2 `commands` there? The output is the same, regardless.

Reply:
Yup, the first command is an ansible plugin, the second is actually running command -v

Reply:
oh ignore me, the first command is to tell `local_action` what it's doing, my bad